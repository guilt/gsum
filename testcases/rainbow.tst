Name: rainbow
Description: Test 2M file with rainbow tile pattern, absolute range
File: rainbow.txt 2M tile 512K 0xaa,0x55
AlgoMatrix: sha1
Range: 1024-1048576
