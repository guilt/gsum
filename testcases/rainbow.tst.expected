Hash-sha1-1024-1048576: 93c35d56b7a2584d39a45a36ce67ae70e6ebade9
